emacs-keybinding-command-tooltip-mode.el

```diff
@@ -100,7 +100,7 @@ commands and variables."
 			 ;; prevent the window from opening, and to eliminate
 			 ;; the minibuffer message.
 			 (describe-variable (intern command))
-		       ;; clear minbuffer
+		       ;; clear minibuffer
```

Comment:
This reminded me that I need to look into word counts. Neither `minbuffer` nor `minibuffer` are in the base dictionary, but it'd be helpful if my tooling hinted at the discrepancy in word hits (it kinda does these days, but I still need to think about how to best improve this....).